src/features/feel-popup/FeelPopupRenderer.js

```diff
@@ -0,0 +1,70 @@
+import { render } from 'preact';
```

Comment:
:spiral_notepad: This module is injected in the parent injector and can be overwritten. This particular one manages popup rendering, and is what will be overwritten in web modeler to implement our custom renderer.